src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,24 @@
+package lotto.view;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class InputView {
+
+    private final static Scanner scanner = new Scanner(System.in);
```

Comment:
상수 선언 시, `static final` 순으로 키워드를 작성해주시면 좋을 것 같습니다.
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```